[2023-10-29T15:41:28.993+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v5.second_task scheduled__2023-01-23T00:00:00+00:00 [queued]>
[2023-10-29T15:41:29.072+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v5.second_task scheduled__2023-01-23T00:00:00+00:00 [queued]>
[2023-10-29T15:41:29.080+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-10-29T15:41:29.343+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): second_task> on 2023-01-23 00:00:00+00:00
[2023-10-29T15:41:29.351+0000] {standard_task_runner.py:57} INFO - Started process 5696 to run task
[2023-10-29T15:41:29.403+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2023-01-23T00:00:00+00:00', '--job-id', '1641', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpwwrxo_5j']
[2023-10-29T15:41:29.503+0000] {standard_task_runner.py:85} INFO - Job 1641: Subtask second_task
[2023-10-29T15:41:29.890+0000] {task_command.py:416} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2023-01-23T00:00:00+00:00 [running]> on host 9bb87dc64328
[2023-10-29T15:41:30.829+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v5' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2023-01-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-23T00:00:00+00:00'
[2023-10-29T15:41:30.838+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-10-29T15:41:30.911+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2023-10-29T15:41:31.192+0000] {subprocess.py:86} INFO - Output:
[2023-10-29T15:41:31.215+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2023-10-29T15:41:31.244+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-10-29T15:41:31.532+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20230123T000000, start_date=20231029T154128, end_date=20231029T154131
[2023-10-29T15:41:31.707+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-29T15:41:32.065+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
